### PR TITLE
Standardised plugin names and jewellery count plugin refactor

### PR DESCRIPTION
**Pull request by honeyhoney**
A bit of housekeeping to standardise the format of plugin names to `<PluginName>Plugin`.

I also thought `jewelry` was a misspelling so I changed all instances of it to `jewellery`. Turns out `jewelry` is just the American English version of `jewellery`. It's spelled `jewellery` in game so I left the change in.

image

### CODE REVIEW

File: runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerTest.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
 	@Mock
 	private SlayerConfig slayerConfig;
 
-	private Slayer slayerPlugin;
+	private SlayerPlugin slayerPlugin;
```
Comment (Adam-) [inline runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerTest.java]: This test should be renamed SlayerPluginTest then

File: runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java [hunk 2]
```diff
@@ -55,7 +55,7 @@
 	@Override
 	protected void startUp() throws Exception
 	{
-		navButton = new NavigationButton("Hiscore", () -> hiscorePanel);
+		navButton = new NavigationButton("HiscorePlugin", () -> hiscorePanel);
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java]: This is the tooltip for the nav button and should not be renamed

File: runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java [hunk 3]
```diff
@@ -39,9 +39,9 @@
 import net.runelite.client.task.Schedule;
 
 @PluginDescriptor(
-	name = "XP Tracker plugin"
+	name = "Xp tracker plugin"
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java]: Keep this as "XP"

File: runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java (hunk 2 above)
Comment (honeyhoney) [reply runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java]: Fixed with fa384b4

File: runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java (hunk 3 above)
Comment (honeyhoney) [reply runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java]: User-facing (or potentially user-facing 'xp' has been changed to 'XP'): 96b4f6e
Class names and such will remain 'Xp' to follow typical Java conventions.

**Pull request merged**